np.sh

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env sh
+
+if test 0 -ne `git status --porcelain | wc -l`; then
```

Comment:
I think this could be expressed a bit cleaner, as `if test -n "$(git status --porcelain)"; then`. No need to count the lines, just check if the response is empty or not.